cmd/generate/Dockerfile.template

```diff
@@ -0,0 +1,16 @@
+# DO NOT EDIT! Generated Dockerfile for {{.main}}.
+FROM {{.builder}} as builder
+
+COPY . .
+
+RUN mkdir -p /var/run/ko && \
```

Comment:
instead of the `make` targets, we will run this (`go build`) directly, rather than via the `Makefile`?

Is that the intention?

Reply:
Yes, this will speed up the process of building the images, since not every image will rebuild every binary as currently done with `make install`